.github/workflows/release.yml

```diff
@@ -0,0 +1,27 @@
+name: Release
+on:
+  push:
+    branches:
+      - main
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: "lts/*"
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+      - name: Build
+        run: npm i -g @vercel/ncc && ncc build src/index.ts
```

Comment:
```suggestion
```

I guess we do not need this part since we use oclif that have a prepack and version script: https://github.com/rap2hpoutre/pg-anonymizer/blob/11e46ce8ac8786d08401da2c27ef7089aa783905/package.json#L46-L48.

So since oclif just need `npm version` and `npm publish` (https://oclif.io/docs/releasing), we do not have to use ncc by vercel (I'm not even sure it's compatible).